components/automate-ui/src/app/pages/client-runs/client-runs.component.ts

```diff
@@ -54,70 +60,86 @@ export class ClientRunsComponent implements OnInit, OnDestroy {
   pageSize = 100;
 
   // The catagories allowed for searching
-  categoryTypes: Chicklet[] = [
+  categoryTypes: SearchBarCategoryItem[] = [
```

Comment:
Updating the original user of the search bar component to use the new additions.